.github/workflows/e2e.yml

```diff
@@ -19,7 +19,7 @@
             }
 
     run-e2e-tests:
-      if: github.event.repository.name != 'terraform-azurerm-avm-template' && github.event.pull_request.head.repo.fork == false
+      if: github.event.pull_request.head.repo.fork == false
```

Comment:
@lonegunmanb could you please explain this change ?

 Is it to allow end to end tests in the template repository ? 

Should not have any effect in this repo, correct ?